helm/ph-ee-engine/operations-importer-rdbms/values.yaml

```diff
@@ -16,6 +16,7 @@ imageTag: ""
 LOGGING_LEVEL_ROOT: "INFO"
 LOGGING_PATTERN_CONSOLE: "%d{dd-MM-yyyy HH:mm:ss.SSS} %magenta([%thread]) %highlight(%-5level) %logger.%M - %msg%n"
 javaToolOptions: "-Xmx256M"
+SPRING_PROFILES_ACTIVE: "local,tenants"
```

Comment:
The tenants config may be different from the one configured in importer rdbms. for example, the connection details are missing here.